ws-nextjs-app/pages/[service]/send/[id]/types.ts

```diff
@@ -63,6 +79,14 @@ export type Field = {
   textArea?: boolean;
 };
 
+export type FieldData = {
+  isValid: boolean;
+  required: boolean;
+  value: OnChangeInputValue;
+  type: string;
```

Comment:
Think this can be typed with `htmlType`

Reply:
Oops sorry @shayneahchoon! I meant it could probably be `type: htmlType`, rather than `string`.